features/javascript/response.feature

```diff
@@ -6,33 +6,33 @@ Feature: Response validation
     """
     const gavel = require('gavel');
     """
-    And you define following HTTP response object:
+    And you define following expected HTTP response object:
     """
-    const real = {
+    const expected = {
       "statusCode": "200",
-      "statusMessage": "OK",
       "headers": {
         "content-type": "application/json",
         "date": "Wed, 03 Jul 2013 13:30:53 GMT",
         "server": "gunicorn/0.17.4",
         "content-length": "30",
         "connection": "keep-alive"
       },
-      "body": "{\n  \"origin\": \"94.113.241.2\"\n}"
+      "body": "{\n  \"origin\": \"94.113.241.2\"\n}",
     };
     """
-    And you define following expected HTTP response object:
+    And you define following HTTP response object:
     """
-    const expected = {
+    const real = {
       "statusCode": "200",
+      "statusMessage": "OK",
```

Comment:
Do we want to have it in the examples if Gavel doesn't validate it?

Reply:
This must be removed.